ADR/Methods/VLM/AVL/configs.avl

```diff
@@ -41,25 +41,25 @@ SECTION
 
 AFILE
 #arquivo_wing_1
-d:\area de trabalho\codigo2ponto0\adr\ADR\Methods\VLM\AVL\airfoils\s1223.dat
+C:\ADR_ReFaP\ADR\ADR\Methods\VLM\AVL\airfoils\s1223.dat
 #==================================================================
 SECTION
 #Xle Yle Zle Chord Ainc Nspanwise Sspace
 
 #Dimensao_wing_secao_2
-0 0.4 0 0.3 0 0 0
+0 2 0 0.3 0 0 0
 #A definition of the second section.
 AFILE
 #arquivo_wing_2
-d:\area de trabalho\codigo2ponto0\adr\ADR\Methods\VLM\AVL\airfoils\s1223.dat
+C:\ADR_ReFaP\ADR\ADR\Methods\VLM\AVL\airfoils\s1223.dat
 #==================================================================
 SECTION
 #Xle Yle Zle Chord Ainc Nspanwise Sspace
 #Dimensao_wing_secao_3
-0 0.8 0 0.2 0 0 0
+0 3.5 0 0.35 0 0 0
 #A definition of the second section.
 AFILE
 #arquivo_wing_3
-d:\area de trabalho\codigo2ponto0\adr\ADR\Methods\VLM\AVL\airfoils\s1223.dat
+C:\ADR_ReFaP\ADR\ADR\Methods\VLM\AVL\airfoils\s1223.dat
 #==================================================================
 

```

Comment:
Please, remove this file change from the branch, as its modifications are automatically done and are not related to this PR.

Reply:
How can i delete the file from branch after it was already modified?

Reply:
and it has been already pushed to the pr.?

Reply:
1. Update your master (checkout to master and pull from origin))
2. Checkout to this branch
3. Rebase this branch on master
4. Reset (mixed) this branch to the base commit (the one where the branch starts from, which will be the last commit on master)

Now all your changes will be on the working are and you can remake the commits from scratch, discarding the changes to the config file, for example.


This is a "history-changing" approach (Dark feelings), but it is good because it makes your final commits as clean and organized as possible.
I usually don't change history just for the master branch, because everything is based on it, but when talking about PR branches, my main goal is clean and organized commits, which when merged, will be easy to verify later if needed.